src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -94,28 +109,96 @@ following to integrate with your `my_flutter` app:
 
 3. Run `pod install`.
 
-Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
-you need to run `flutter pub get` from `some/path/my_flutter` to refresh the list
+{{site.alert.note}}
+When you change the Flutter plugin dependencies in `my_flutter/pubspec.yaml`,
+run `flutter pub get` in your Flutter module directory to refresh the list
 of plugins read by the `podhelper.rb` script. Then run `pod install` again from
-`some/path/MyApp`.
+in your application at`some/path/MyApp`.
+{{site.alert.end}}
```

Comment:
Do you have to do this if you `flutter upgrade` or `flutter channel` too?

Reply:
I don't think upgrade or channel would alter anything in .flutter-plugins, but I can add it if you know something I don't.

Reply:
I'm just guessing. Wouldn't this change the engine framework which gets copied over to your .ios folder if you ran flutter packages get? Don't know if you need a subsequent pod install to make things match.

Reply:
The right version of the Flutter.framework should get copied the next time they run their app via the `xcode_backend` script.